tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -16,6 +16,9 @@ Login To ODH Dashboard
    ${authorize_service_account} =  Is odh-dashboard Service Account Authorization Required
    Run Keyword If  ${authorize_service_account}  Authorize odh-dashboard service account
 
+Wait for ODH Dashboard to Load
+  Wait For Condition  return document.title == "Red Hat OpenShift Data Science Dashboard"
```

Comment:
Can you move `Red Hat OpenShift Data Science Dashboard` to a variable? This will make it easier to update when/if the title changes when running this against upstream ODH

Reply:
Done in 1d4c0a7